### ISSUE TITLE
gobo linux is impossible to install on m.2 nvme drives

### ISSUE THREAD

**Issue by MrKorner** [gobolinux/Installer#14]
I am not sure why this is but as the title says, gobo linux installer just cannot see nvme ssd. i would like to provide more info but i dont really have much to say here.

**Comment by lucasvr** [comment]
Can you show the output of `parted -l`? What does the Installer have to say about the possible installation targets?

**Comment by hishamhm** [comment]
I believe I can help with testing that too. Now that I recall, I had to
hack the Installer sources to make 016 install on my nvme. I even thought I
had sent the patches. IIRC it was a matter of making it recognize the
proper /dev name and not much else.

>

**Comment by hishamhm** [comment]
@MrKorner did you try 016 or the latest 017alpha?

**Comment by MrKorner** [comment]
i did not tried A17 and i wont be able to try show parted -l because rn i
am settled on other distro but if an opportunity comes up ill see if i can
try that

**Comment by lucasvr** [comment]
No worries. I will investigate this issue on QEMU.

**Comment by hishamhm** [comment]
@lucasvr this is `parted -l` on my machine:

```
Model: Unknown (unknown)
Disk /dev/nvme0n1: 1024GB
Sector size (logical/physical): 512B/512B
Partition Table: gpt
Disk Flags: 

Number  Start   End     Size    File system  Name                          Flags
 1      1049kB  274MB   273MB   fat32        EFI system partition          boot, hidden, esp
 2      274MB   290MB   16.8MB               Microsoft reserved partition  msftres
 3      290MB   1023GB  1023GB  ext4         Basic data partition          legacy_boot
 4      1023GB  1024GB  1049MB  ext4
```

**Comment by hishamhm** [comment]
```
ls -l /dev/nvm*
crw------- 1 root root 239,  0 May  3 14:46 /dev/nvme0
brw-rw---- 1 root disk 259,  0 May  4 19:43 /dev/nvme0n1
brw-rw---- 1 root disk 259,  1 May  4 19:43 /dev/nvme0n1p1
brw-rw---- 1 root disk 259,  2 May  4 19:43 /dev/nvme0n1p2
brw-rw---- 1 root disk 259,  3 May  4 19:43 /dev/nvme0n1p3
brw-rw---- 1 root disk 259,  4 May  4 19:43 /dev/nvme0n1p4
crw------- 1 root root  10, 59 May  3 14:46 /dev/nvme-fabrics
```

**Comment by lucasvr** [comment]
Fixed by commit 49d290ace660630dced255bc66e112d662dd1752
